ai21/clients/studio/resources/chat/chat_completion.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import List, Optional, Union, Any, Dict
+
+from ai21.clients.studio.resources.studio_resource import StudioResource
+from ai21.models.chat import ChatMessage, ChatCompletionResponse
+from ai21.types import NotGiven, NOT_GIVEN
+from ai21.utils.typing import remove_not_given
+
+__all__ = ["ChatCompletions"]
+
+
+class ChatCompletions(StudioResource):
+    _module_name = "chat/complete"
+
+    def create(
+        self,
+        model: str,
+        messages: List[ChatMessage],
+        n: Optional[int] | NotGiven = NOT_GIVEN,
+        logprobs: Optional[bool] | NotGiven = NOT_GIVEN,
+        top_logprobs: Optional[int] | NotGiven = NOT_GIVEN,
+        max_tokens: Optional[int] | NotGiven = NOT_GIVEN,
+        temperature: Optional[float] | NotGiven = NOT_GIVEN,
+        top_p: Optional[float] | NotGiven = NOT_GIVEN,
+        stop: Optional[Union[str, List[str]]] | NotGiven = NOT_GIVEN,
+        frequency_penalty: Optional[float] | NotGiven = NOT_GIVEN,
+        presence_penalty: Optional[float] | NotGiven = NOT_GIVEN,
```

Comment:
What happens if None is passed?

Reply:
@Asaf-m Awesome. Thanks for the heads up